app/controllers/glossary_terms_controller.rb

```diff
@@ -99,6 +103,28 @@ def show_past
 
   # --------- Other private methods
 
+  def reload_form(form)
+    add_model_error_messages_to_flash
+    init_image_form_instance_variables
+    render(form)
+  end
+
+  def add_model_error_messages_to_flash
+    model_error_messages = @glossary_term.errors.messages.values.join("\n")
+    flash_error(model_error_messages)
+  end
+
+  def init_image_form_instance_variables
+    @copyright_holder ||= @user.name
+    @copyright_year ||= Time.now.utc.year
+    @upload_license_id ||= @user.license_id
+    # rubocop:disable Naming/MemoizedInstanceVariableName
+    # RuboCop wants to name the ivar "@update_image_form_instance_variables"
+    @licenses ||= License.current_names_and_ids(@user.license)
```

Comment:
Is there a better way to resolve the RuboCop complaint about line 123:
>Memoized variable @licenses does not match method name init_image_form_instance_variables. Use @init_image_form_instance_variables instead.

(I'm not going to name the ivar `@init_image_form_instance_variables`.)

Reply:
Why are you memoizing these values anyway?  Specifically what is the `||=` getting you that a simple `=` wouldn't?  I don't see anywhere obvious that these values are getting setting twice.  If you can remove the `||` from any of these and then put it at the end, I believe the violation will go away.

Reply:
Thank you.
You're absolutely right.
I un-memoized the ivars. Also improved them by re-using the values input by the user (rather than initializing) when reloading the form.